src/App/ComponentsDocumentation/components/Badge/index.js

```diff
@@ -2,28 +2,40 @@ import React from "react";
 
 import { ComponentPreview, DocContainer } from "@docutils";
 import CodeTags from "@components/CodeTags";
+import {badgeShowcase} from "./constants"
 
 const Overview = () => (
     <>
         <h2 id="overview">Overview</h2>
-        <p>Use class <CodeTags type="secondary" code=".badge" /> along with <CodeTags type="secondary" code=".badge-{color}" /> to display the badge of your choosing.</p>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <p>Badge <span className="badge badge-default">Default</span></p>
-            <p>Badge <span className="badge badge-inactive">Inactive</span></p>
-            <p>Badge <span className="badge badge-warning">Warning</span></p>
-        </ComponentPreview>
-    </>
-);
+        <p>Use class <CodeTags type="secondary" code=".badge" /> along with <CodeTags type="secondary" code=".badge-{color}" /> to display the badge of your choosing. Badges are often used with numbers. Use <CodeTags type="secondary" code=".badge-number" /> when a badge suited for use with numbers is desired.</p>
```

Comment:
Weird sentence.. At least the last part
Maybe: Use <CodeTags type="secondary" code=".badge-number" /> when a badge with only numbers is desired.